QRCoder/SvgQRCode.cs

```diff
@@ -285,7 +285,7 @@ public SvgLogo(Bitmap iconRasterized, int iconSizePercent = 15, bool fillLogoBac
                     using (var bitmap = new Bitmap(iconRasterized))
                     {
                         bitmap.Save(ms, System.Drawing.Imaging.ImageFormat.Png);
-                        _logoData = Convert.ToBase64String(ms.GetBuffer(), Base64FormattingOptions.None); 
+                        _logoData = Convert.ToBase64String(ms.GetBuffer(), 0, (int)ms.Length, Base64FormattingOptions.None); 
```

Comment:
`GetBuffer()` returns the current working memory buffer allocated by `MemoryStream`, including extra padding that is reserved but not yet used by the `MemoryStream`.  This is unlike `ToArray` which copies the working data to a new array of the correct length.  To properly use `GetBuffer`, the length must be passed to `Convert.ToBase64String` so that only the working data is encoded.